applications/dashboard/models/class.usermodel.php

```diff
@@ -1173,16 +1196,13 @@ public function expandUsers(array &$rows, array $columns, array $options = []) {
                         // Add an alias to Photo. Currently only used in API calls.
                         setValue('PhotoUrl', $user, $photo);
                     } else {
-                        $user = [
-                            'userID' => 0,
-                            'name' => 'unknown',
-                            'email' => 'unknown@example.com'
-                        ];
-                        $user['photoUrl'] = self::getDefaultAvatarUrl($user);
```

Comment:
User was always undefined here anyways so not passing the parameter in `getUnknownFragment()` has no impact.